<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintDialog</class>
 <widget class="QDialog" name="PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Selection</string>
  </property>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>231</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Print visible glypfs</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonPrintScreen">
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Print all the glypfs</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonPrintAll">
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
